project4/styles.css

```diff
@@ -0,0 +1,442 @@
+}
+
+.logo {
+    left: 0;
+    top: 30px;
+    width: 35%;
+}
+
+.theme {
+    color: var(--color-white);
+    font-size: 8px;
+    letter-spacing: 3px;
+    line-height: 1;
+}
+
+
+
+
+.main-title {
+    padding-bottom: 80px;
+    padding-left: 30px;
+    padding-top: 80px;
+}
+
+.main-title p {
+    color: var(--color-pink);
+    font-family: bigsmalls-bold, sans-serif;
+    font-size: 60px;
+    line-height: 25px;
+}
+
+.first {
+    border: 1.5px solid var(--color-pink);
+}
+
+.speaker {
+    border-top-left-radius: 30%;
+    border-bottom-right-radius: 30%;
+    width: 100%;
+
+}
+
+.speaker-container {
+    display: block;
+    margin-top: 50px;
+    margin-bottom: 50px;
+    width: 80%;
+}
+
+
+.ticket-type p,
+.ticket-type a {
+    display: inline-block;
+    vertical-align: middle;
+    width: 45%;
+}
+
+.ticket-type a {
+    background-color: var(--color-white);
+    border-radius: 50%;
+    color: var(--color-black);
+    font-family: bigsmalls-bold, sans-serif;
+    font-size: 15px;
+    margin-left: 18px;
+    line-height: 15px;
+    padding-bottom: 55px;
+    padding-top: 55px;
+    text-align: center;
+    text-decoration: none;
+    width: 130px;
+}
+
+.ticket-type p {
+    margin: 0;
+}
+
+.clearfix::after {
+    clear: both;
+    content: "";
+    display: table;
+}
+
+
+
+
+
+.share-list {
+    letter-spacing: 30px;
+    list-style: none;
+    margin: auto;
+    text-align: center;
+
+}
+
+.share-list a {
+    display: block;
+    margin: auto;
+}
+
+.share-list svg {
+    height: 40px;
+    width: 40px;
+}
+
+.share-list li {
+    display: inline-block;
+}
+
+
+
+
+.share-list a:hover path {
+    fill: var(--color-white);
+}
+
+.share-text {
+    margin: auto;
+    padding: 50px;
+    text-align: center;
+}
+
+.share-text h2 {
+    font-size: 40px;
+}
+
+
+
+@media (min-width: 630px) {
+
+    .speaker-container {
+        display: inline-block;
+        vertical-align: top;
+        width: 27%;
+    }
+
+    .speaker-container+.speaker-container {
+        margin-left: 28px;
+    }
+
+}
+
+@media (min-width: 920px) {
+
+    .exhibits {
+        display: inline-block;
+        width: 20%;
+
+    }
+
+    .schedule-wrapper {
+        display: flex;
+        align-items: center;
+        margin: auto;
+        padding-left: 5%;
+        padding-right: 5%;
+    }
+
+    .schedule h3 {
+        margin-left: 200px;
+    }
+
+    .schedule-day-one,
+    .schedule-day-two {
+        display: inline-grid;
+        font-size: 12px;
+        grid-template-columns: auto auto;
+        grid-gap: 10px;
+        grid-column-gap: 50px;
+        width: 500px;
+
+    }
+
+    section#schedule {
+        display: block;
+        margin: auto;
+    }
+
+    section#schedule h2 {
+        letter-spacing: 20px;
+        text-align: center;
+    }
+
+
+    .schedule+.schedule {
+        padding-left: 50px;
+    }
+
+    .schedule {
+        display: inline-block;
+    }
+
+    .exhibitions-wrapper {
+        display: grid;
```

Comment:
🏆 Congrats on starting to learn grid. It's pretty amazing. Continue your learning with [Learn CSS Grid](https://learncssgrid.com/) and [these](https://www.gridcritters.com/) [games](https://cssgridgarden.com/) to test your understanding.